github-actions/setup-bazel-remote-exec/set_bazelrc_for_remote_exec.sh

```diff
@@ -0,0 +1,4 @@
+echo """
+build --config=remote
+build --repository_cache=$HOME/bazel_repository_cache
```

Comment:
I honestly was thinking we'd have a totally separate action where we do this. Like `setup_bazel` and then that composite action would call the setup RBE action, and later on also add this to the bazelrc.

This action is scoped to remote execution- and repository caching is unrelated to RBE

Reply:
Are there cases where we don't want to set up remote execution?

Reply:
Well this action sets up remote execution- but repository caching has nothing to do with it

Reply:
True, what if this action is converted into "setting up bazel for CI" and encompasses both things?

Reply:
I'd prefer to keep it separate as an individual building block- especially because it's already non-trivial (and intentionally a little obfuscated)